variables.tf

```diff
@@ -241,6 +241,12 @@ variable "cloud_functions_region_map" {
   }
 }
 
+variable "cloud_function_image_prefix" {
```

Comment:
I thought about it, I think that `cloud_run_image_prefix` is more clear, this one might be a bit confusing

Reply:
Good idea, thanks! Implemented.